tempesta_fw/cache.c

```diff
@@ -345,7 +345,7 @@ tfw_cache_employ_req(TfwHttpReq *req)
 		req->cache_ctl.flags |= TFW_HTTP_CC_CFG_CACHE_BYPASS;
 		return false;
 	}
-	/* cache_fulfill - work as usual in cache mode. */
+	/* cache_fulfil - work as usual in cache mode. */
```

Comment:
It's spelled `fulfill` everywhere else in the code, including constant names and even configuration parameter: `cache_fulfill`. As far as I understand, both `fulfil` and `fulfill` are accepted. We just need to use either one consistently.

Reply:
Yes, American/British English difference. This is the only mistakenly "fixed" spelling, reverted it. Thank you!